Gems/Atom/RHI/Vulkan/Code/Source/RHI/BufferView.h

```diff
@@ -25,7 +25,7 @@ namespace AZ
         public:
             using ResourceType = Buffer;
 
-            AZ_CLASS_ALLOCATOR(BufferView, AZ::ThreadPoolAllocator, 0);
+            AZ_CLASS_ALLOCATOR(BufferView, AZ::SystemAllocator, 0);
```

Comment:
Consider leaving a comment to explain the choice of SystemAllocator, in case someone tries to change this back to a ThreadPoolAllocator in the future without knowing the history.

Reply:
Will do